development/conf.py

```diff
@@ -159,7 +169,11 @@
 #html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
-#html_sidebars = {}
+html_sidebars = {
+    '**': [
+        'versioning.html',
```

Comment:
Is this right? Or is it versions.html ?

Reply:
Oh, nvmnd, I think this is right. Seems to be working.

Reply:
I think it's actually not needed as the rtd theme does not support sidebards. 🤷‍♂️